### PR TITLE
[meta] Update list of supported platforms

### PR DESCRIPTION
**Pull request by olivielpeau**
In particular, use valid versions of opensuse
(based on official Ansible list available at
https://galaxy.ansible.com/api/v1/platforms/?name__icontains=opensuse) and SLES

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -16,20 +16,27 @@ galaxy_info:
...
         - 6
     - name: opensuse
       versions:
-        - 11
-        - 12
+        - 12.1
+        - 12.2
+        - 12.3
+        - 13.1
+        - 13.2
```
Comment (KSerrania) [inline meta/main.yml:31]: Should we also add the SLES versions we support (11SP3, 11SP4, 12, 12SP1, 15)?

Comment (olivielpeau) [reply meta/main.yml:31]: oh, nice find, didn't see `SLES` is regarded as a separate platform altogether. I'll update 👍

**Review by KSerrania — APPROVED** [review]

**Review by hush-hush — APPROVED** [review]

**Comment by hush-hush** [comment]
@olivielpeau Do you mind updating the CHANGELOG too ?

**Comment by olivielpeau** [comment]
@hush-hush added a release note (didn't put a version number since I don't know what the next version will be :) )

**Pull request merged**